index.html

```diff
@@ -334,6 +335,21 @@ <h2 class="languages-title ">Skills</h2>
 
       </section>
 
+      <section class="contact-me-bg">
+        <div class="contact-me">
+          <h2 class="contact-me-title-footer">Contact Me</h2>
+            <p class="contact-me-description">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+            </p>
+            <form class="contact-me-form" action="https://formspree.io/f/xoqbvvly" method="POST">
+              <input type="name" name="name" placeholder="full name" class="contact-me-input" maxlength="30" required>
```

Comment:
- [ ] kindly update your input type which should be `text` and not `name` , after that your project will be ready , great job 👏

Reply:
I have just updated it, can you please check again.